src/file_manager.py

```diff
@@ -11,37 +11,111 @@
 
 
 class FileManager:
-    def __init__(self, fstring: str, field_names: tuple,
-                 bytes_per_file: int = 1000, files_per_folder: int = 100) -> None:
+    """
+    Deals with reading and writing to the files.
+    """
+
+    @staticmethod
+    def unpack_db_info(data: bytes, isBigEndian: bool = False):
```

Comment:
Not sure if it's worth the time, but may be worth putting the version first.
It will make it drastically easier to identify the version of the database in the future.